### PR TITLE
Fix delete old rules based on vlan stats

### PR DESCRIPTION
**Pull request by durgakako**

### CODE REVIEW

File: src-java/nbworker-topology/nbworker-storm-topology/src/test/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsServiceTest.java [hunk 1]
```diff
@@ -229,112 +227,6 @@ public void strictBandwidthUpdateTest() throws FlowNotFoundException, InvalidFlo
...
-                .flowId(FLOW_ID_1)
-                .vlanStatistics(new HashSet<>())
-                .source(buildPathEndpoint(newSrcVlan))
-                .destination(buildPathEndpoint(newDstVLan))
-                .build();
-
-        Flow updatedFlow = flowOperationsService.updateFlow(new FlowCarrierImpl(), receivedFlow);
-        assertTrue(updatedFlow.getVlanStatistics().isEmpty());
-    }
-
```
Comment (niksv) [inline src-java/nbworker-topology/nbworker-storm-topology/src/test/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsServiceTest.java:337]: But why did you remove all tests?

File: src-java/nbworker-topology/nbworker-storm-topology/src/test/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsServiceTest.java [hunk 2]
```diff
@@ -229,112 +227,6 @@ public void strictBandwidthUpdateTest() throws FlowNotFoundException, InvalidFlo
...
-        expectedVlanStatistics.add(31);
-
-        FlowPatch receivedFlow = FlowPatch.builder()
-                .flowId(testFlowId)
-                .vlanStatistics(expectedVlanStatistics)
-                .build();
-
-        Flow updatedFlow = flowOperationsService.updateFlow(new FlowCarrierImpl(), receivedFlow);
-
-        assertThat(updatedFlow.getVlanStatistics(), containsInAnyOrder(expectedVlanStatistics.toArray()));
```
Comment (niksv) [inline src-java/nbworker-topology/nbworker-storm-topology/src/test/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsServiceTest.java:256]: You should not delete this test, but change it. This test must check that `updateFlow` would NOT change the vlan stats in the flow

**Review by niksv — CHANGES_REQUESTED** [review]

File: src-java/nbworker-topology/nbworker-storm-topology/src/test/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsServiceTest.java (hunk 2 above)
Comment (durgakako) [reply src-java/nbworker-topology/nbworker-storm-topology/src/test/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsServiceTest.java:256]: Updated.

**Review by niksv — APPROVED** [review]

**Review by pablomuri — APPROVED** [review]
